src/sentry/metrics/base.py

```diff
@@ -78,3 +78,18 @@ def distribution(
         stacklevel: int = 0,
     ) -> None:
         raise NotImplementedError
+
+    def event(
+        self,
+        key: str,
+        value: str,
+        alert_type: str | None = None,
+        aggregation_key: str | None = None,
+        source_type_name: str | None = None,
+        date_happened: int | None = None,
```

Comment:
Verified that `date_happened` will default to system time in case it is not passed. [Docs](https://datadogpy.readthedocs.io/en/latest/#datadog.api.Event).
So removed it.